tests/unit/lms/events/test_event.py

```diff
@@ -34,15 +31,6 @@ def test_lti_event(
         )
         assert event.assignment_id == assignment_service.get_assignment.return_value.id
 
-    @pytest.mark.usefixtures("lti_role_service", "course_service", "assignment_service")
-    def test_lti_event_when_no_application_instance(
```

Comment:
This was forcing the mock to behave differently than the code (get_current returning None).

So far our LTIEvents imply that they are attached to an instance.